### PR TITLE
Added SESSION_SECRET environmental variable to sample.env

### PR DESCRIPTION
**Pull request by patelg123**
#### Pre-Submission Checklist
- [ ] Your pull request targets the `develop` branch.
- [ ] Branch starts with either `fix/` or `feature/`
- [ ] All new and existing tests pass the command `npm test`.
- [ ] Any new files are included in the grunt tasks

#### Type of Change
- [ ] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)

#### Checklist:
- [ ] Tested changes locally.
- [ ] New tests added?
- [ ] Closes currently open issue (replace XXXX with an issue no): Closes #XXXX

#### Full Description of your Changes

### CODE REVIEW

File: sample.env [hunk 1]
```diff
@@ -13,5 +13,7 @@ MONGODB_URI='mongodb://localhost:27017/fiddles'
 
 
 # PRIVATE section:-
+SESSION_SECRET='&E)H@McQfTjWnZr4'
```
Comment (joshghent) [inline sample.env]: Maybe change this to something like `keyboard-cat` as an example this looks to be more like a password that someone is actually using.

**Review by joshghent — CHANGES_REQUESTED** [review]
Hi mate, thanks for updating this! Could you please update the docs to add that you need to generate a random phrase for the session secret and they can do that by going here: https://www.random.org/passwords/?num=1&len=10&format=html&rnd=new

File: sample.env (hunk 1 above)
Comment (patelg123) [reply sample.env]: @joshghent Please can you take a look now.

**Review by joshghent — APPROVED** [review]
Great work mate, thanks for this! 🎉

**Pull request merged**